### ISSUE TITLE
Add Uploaded File(Layer) Name to GeoLibrary uploader tool

### ISSUE THREAD

**Issue by digitalkg** [culibraries/geoDataLoader#16]
Is it possible to add the name of the file that was uploaded as a field that shows in the loader? Right now it shows title, status, and description. Having this would be helpful if we have to clean out extraneous files from the GeoServer.

**Comment by mbstacy** [comment]
Yes, think this is a good idea. Hopefully, get this done within the next Month.
